src/main/java/org/agrona/concurrent/errors/ErrorLogReader.java

```diff
@@ -70,7 +71,7 @@ public static int read(final AtomicBuffer buffer, final ErrorConsumer consumer,
                     buffer.getInt(offset + OBSERVATION_COUNT_OFFSET),
                     buffer.getLong(offset + FIRST_OBSERVATION_TIMESTAMP_OFFSET),
                     lastObservationTimestamp,
-                    buffer.getStringUtf8(offset + ENCODED_ERROR_OFFSET, length - ENCODED_ERROR_OFFSET));
+                    buffer.getStringUtf8(offset + ENCODED_ERROR_OFFSET - BitUtil.SIZE_OF_INT /* remove length prefix */, length - ENCODED_ERROR_OFFSET));
```

Comment:
it would be better to use getStringUtf8WithoutLength instead of adjusting the offset.